pepper-apis/src/main/java/org/broadinstitute/ddp/event/pubsubtask/api/PubSubTaskLogUtil.java

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.ddp.event.pubsubtask.api;
+
+/**
+ * Adds a prefix to log messages in PubSubTask API.
+ */
+public class PubSubTaskLogUtil {
```

Comment:
I'm not sure we need this?  logback will include the class/method names, which should give us enough information to track this to pubsub.

Reply:
It's easier to find log messages related only to this api: `textPayload: "PUBSUB_TASK"`